composer.json

```
@@ -18,11 +18,11 @@
         "migrify/php-config-printer": "^0.3.31",
         "nette/robot-loader": "^3.2",
         "nette/utils": "^3.1",
-        "nikic/php-parser": "4.9.0",
+        "nikic/php-parser": "dev-master",
```

it seems needs a hash to avoid random error: 

```suggestion
        "nikic/php-parser": "dev-master#88be6127fa91145ebf4f02a75e4e813f09d7c99c",
```

---

This is on purpose, it won't be merged like this.

See https://github.com/nikic/PHP-Parser/issues/714#issuecomment-696968632